portfolio/src/main/webapp/index.html

```diff
@@ -22,40 +23,166 @@
+                <h2 style="color:aliceblue">Why do I like programming</h2>
+                <p style="color:aliceblue; font-size:15px">I am a coding enthusiast that finds the programming world as being one of the most fascinating things that exist, because of the fact that it is one of the very few context in which you can create your own universe, establish your very own rules and let your creativity run free and express your mind through all the lines of code. Also it is an area in which you meet extraordinary people and you get the chance to develop products that really have a positive impact on society. Through computer science you can be a life-changer for somebody. You know, not all heroes wear capes.</p>
+                </div>
+                <br></br>
+                <div class="column" style="background-color:teal">
+                <h2 style="color:aliceblue">Hobbies</h2>
+                <p style="color:aliceblue; font-size:15px">My hobbies are reading, writing, dancing, listening to music, watching TV series, traveling and learning new languages.</p>
+                </div>
+            </div>
+        </section> 
+        <br></br>
+        <section class="section-projects-skills" id="section-projects-skills">
+            <h1 class="section-projects-title" style="color: aliceblue; font-size: 20px; background-color:cadetblue">Projects</h1>
+            <br></br>
+            <div class="projects">
+            <div class="project" style="background-color:teal">
+                <h2 style="color:aliceblue;">Smart Calendar</h2>
+                <p style="color:aliceblue; font-size:15px;">It is a website where you can check what task do you need to complete and which day, designed with javascript, ajax and node.js</p>
+                <a href="https://www.github.com/AlexandraBulaceanu/Proiect-TW--SMART-CALENDAR">Check it out!</a>
+            </div>
+            <br></br>
+            <br></br>
+            <br></br>
+            <div class="project" style="background-color:teal">
+                <h2 style="color:aliceblue">Supplied</h2>
+                <p style="color:aliceblue; font-size:15px;">Supplied is a fin-tech application that is specialized in finacial forecasting, planning and budget administration. An ideal shopping list is generated to meet the user's needs.<br></br>It was developed with Python, Javascript, MongoDB, Flask. Together with my team we won prizes at the following hackathons: IT Fest, HackTheVirus and Start-up Weekend.  </p>
+                <a href="https://www.devpost.com/software/food-tracker-tnhe94">Check it out!</a>
             </div>
-            <div class="education">
-            <h2>Education<h2>
-            <p>I graduated from National College of Computer Science "Tudor Vianu" and I am now a student that has just finised the first year at uni. I have completed numerous computer science courses during this time.<p>
+            <br></br>
```

Comment:
Generally, many <br> tags is a bad practice. Can you try to set margin-top parameter with a <p> tag if you want to separate to paragraph?

Reply:
Thank you! I have changed it. I am making a new pull request.